### PR TITLE
Added iOS example for add-to-app Pigeon

### PR DESCRIPTION
**Pull request by gaaclarke**
fixes https://github.com/flutter/flutter/issues/52777

### CODE REVIEW

File: add_to_app/books/README.md [hunk 1]
```diff
@@ -18,13 +18,15 @@ page.
   * On Android, the Kotlin app already uses GSON and OkHttp for networking and
     references the Google Books API as a data source. These same libraries also
     underpin the data fetched and shown in the Flutter screen.
-  * iOS TODO.
+  * On iOS, the Swift app does a similar fetch and parse of the Google Books API
+    using iOS's builtin libraries.
```
Comment (RedBrogdon) [inline add_to_app/books/README.md:22]: nit: "built-in"

File: add_to_app/books/ios_books/IosBooks/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import Flutter
```
Comment (RedBrogdon) [inline add_to_app/books/ios_books/IosBooks/AppDelegate.swift:1]: nit: copyright header needed.

File: add_to_app/books/ios_books/IosBooks/SceneDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import UIKit
```
Comment (RedBrogdon) [inline add_to_app/books/ios_books/IosBooks/SceneDelegate.swift:1]: Is this one of your files or an auto-generated one? If it's the former, add a copyright header.

File: add_to_app/books/ios_books/IosBooks/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,127 @@
+import Flutter
```
Comment (RedBrogdon) [inline add_to_app/books/ios_books/IosBooks/ViewController.swift:1]: nit: copyright header.

**Review by RedBrogdon — COMMENTED** [review]
Just some nits.

One real question, though: can this be added to the iOS CI script in `/tool`?

File: add_to_app/books/README.md (hunk 1 above)
Comment (gaaclarke) [reply add_to_app/books/README.md]: done

File: add_to_app/books/ios_books/IosBooks/AppDelegate.swift (hunk 2 above)
Comment (gaaclarke) [reply add_to_app/books/ios_books/IosBooks/AppDelegate.swift:5]: done

File: add_to_app/books/ios_books/IosBooks/SceneDelegate.swift (hunk 3 above)
Comment (gaaclarke) [reply add_to_app/books/ios_books/IosBooks/SceneDelegate.swift:5]: done

File: add_to_app/books/ios_books/IosBooks/ViewController.swift (hunk 4 above)
Comment (gaaclarke) [reply add_to_app/books/ios_books/IosBooks/ViewController.swift:5]: done

**Comment by gaaclarke** [comment]
> One real question, though: can this be added to the iOS CI script in `/tool`?

Done, added for iOS.  The flutter module is already covered.

**Review by RedBrogdon — APPROVED** [review]

File: add_to_app/books/README.md [hunk 5]
```diff
@@ -36,10 +38,13 @@ page.
   * If the `schema.dart` is modified, the generated classes can be updated with
 
       ```bash
-      flutter pub run pigeon \
-          --input pigeon/schema.dart \
-          --java_out ../android_books/app/src/main/java/dev/flutter/example/books/Api.java \
-          --java_package "dev.flutter.example.books"
+      flutter pub run pigeon --input pigeon/schema.dart \
```
Comment (xster) [inline add_to_app/books/README.md:41]: Would be nice to bring this into the config function so it's easy to generate (and harder to make mistakes) for someone who just downloaded this, changed a line and want to try again.

**Review by xster — COMMENTED** [review]
Thanks for adding this!

File: add_to_app/books/ios_books/IosBooks.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -0,0 +1,463 @@
...
+		};
+		0DDE4C4D25E0390700389930 /* Release */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 18154B6C0196BC72EA210903 /* Pods-IosBooks.release.xcconfig */;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = 5B6X25244C;
```
Comment (xster) [inline add_to_app/books/ios_books/IosBooks.xcodeproj/project.pbxproj:425]: let's remove these

**Review by xster — COMMENTED** [review]
I couldn't run these locally somehow. I'm getting `[VERBOSE-2:shell.cc(242)] Dart Error: Can't load Kernel binary: Invalid kernel binary format version.` on simulator and phone.

**Pull request merged**